editor/editor_help.cpp

```diff
@@ -3949,6 +3981,17 @@ void EditorHelpBit::_meta_clicked(const String &p_select) {
 		} else {
 			_go_to_help(topic + ":" + symbol_class_name + ":" + link);
 		}
+	} else if (p_select.begins_with("open-ext")) {
+		String path = ProjectSettings::get_singleton()->globalize_path(p_select.substr(9));
+		OS::get_singleton()->shell_show_in_file_manager(path, true);
+	} else if (p_select.begins_with("open")) {
+		if (help_data.doc_type.type == "PackedScene") {
+			EditorNode::get_singleton()->load_scene(p_select.substr(5));
+		} else {
+			EditorNode::get_singleton()->load_resource(p_select.substr(5));
+		}
+	} else if (p_select.begins_with("show")) {
+		FileSystemDock::get_singleton()->navigate_to_path(p_select.substr(5));
```

Comment:
I would suggest using a colon to more clearly denote a "pseudo-protocol" (like `open-res:`). Historically we use the symbols (`$`, `#`, `@`, `^`), but there are also real protocols `http:` and `https:`.